web/crux/templates/self-hosted-gitlab.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Self-managed GitLab",
+  "description": "This template contains Self-managed GitLab, deploy and maintain your own GitLab instance.",
+  "registries": [
+    {
+      "name": "Docker Hub Gitlab",
+      "type": "hub",
+      "hub": {
+        "imageNamePrefix": "gitlab"
+      }
+    }
+  ],
+  "images": [
+    {
+      "name": "Gitlab self-managed",
+      "registryName": "Docker Hub Gitlab",
+      "image": "gitlab-ce",
```

Comment:
Is this the official image?

Reply:
Yes, it is the community edition of GitLab.
https://gitlab.com/rluna-gitlab/gitlab-ce